pkg/storaged/side-panel.jsx

```diff
@@ -117,7 +138,7 @@ class SidePanelRow extends React.Component {
 
         return (
             <FlexItem data-testkey={this.props.testkey}
-                      className="sidepanel-row"
+                      className={"sidepanel-row" + (this.props.className ? " " + this.props.className : "")}
```

Comment:
It doesn't hurt to move the space to the initial `"side-panel-row "` string, and simplify this.

Reply:
done